app/src/main/res/layout/activity_main.xml

```diff
@@ -10,35 +10,35 @@
         android:id="@+id/mail_usuario"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="8dp"
-        android:layout_marginLeft="8dp"
-        android:layout_marginBottom="13dp"
+        android:layout_marginStart="@dimen/mail_margin_horizontal"
+        android:layout_marginLeft="@dimen/mail_margin_horizontal"
+        android:layout_marginBottom="@dimen/mail_bottom"
         android:ems="10"
         android:hint="@string/exampleMail"
         android:inputType="textEmailAddress"
         android:singleLine="false"
         app:layout_constraintBottom_toTopOf="@+id/password"
-        app:layout_constraintStart_toEndOf="@+id/usuario_text" />
+        app:layout_constraintStart_toEndOf="@+id/etiqueta_mail" />
 
     <EditText
         android:id="@+id/password"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="8dp"
-        android:layout_marginLeft="8dp"
-        android:layout_marginBottom="18dp"
+        android:layout_marginStart="@dimen/pass_margin_horizontal"
+        android:layout_marginLeft="@dimen/pass_margin_horizontal"
+        android:layout_marginBottom="@dimen/pass_bottom"
         android:ems="10"
         android:inputType="textPassword"
         app:layout_constraintBottom_toTopOf="@+id/log_in"
         app:layout_constraintStart_toEndOf="@+id/etiqueta_password" />
 
     <TextView
-        android:id="@+id/usuario_text"
+        android:id="@+id/etiqueta_mail"
         android:layout_width="100sp"
```

Comment:
use dimens please, and use sp only for strings (textSize), in the rest you can use dp

Reply:
done